base/include/H264Metadata.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include "FrameMetadata.h"
+
+class H264Metadata : public FrameMetadata
```

Comment:
Not clear why we need this? It is called K264Metadata but it only is adding Width/Height to FrameMetadata which is needed in many other metadata as well

Reply:
Idea was to have a bare minimum child class of FrameMetadata for H264 metadata like we have RawImage and RawPlanar , other members of H264 props could be further included as per need basis.